### ISSUE TITLE
[Suggestion] Auto extract QSD

### ISSUE THREAD

**Issue by CodexAdrian** [lorddusk/HQM#665]
A version of the QSD that has a smaller form that, when placed on the side of the block, will automatically try to extract items and fluids (and energy if you like the energy task idea). This would be ideal when you are trying to submit tasks that require quite a large number of items, you could stick it onto the side of a chest or fluid tank and it would automatically pull what it could from the storage. 

(If you like this idea, I will submit a PR with this implemented)

**Comment by kirderf1** [comment]
So while the normal QSD would accept insertions, this block would actively extract things from a specific direction?

**Comment by CodexAdrian** [comment]
Yes, it would stick to a side of an inventory and try to extract things from it automatically, like a little leech

**Comment by canitzp** [comment]
@CodexAdrian This ticket is now nearly one year old. Is this feature still desired for you? If so feel free to open a pr, otherwise I'll let this ticket stay open, so we can track it.
